Сетка страниц и крупных блоков на флексах

KaterinaOsch opened the pull request:
Та часть, которую успела сделать. 

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/33/user/1923813/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/1923813-sedona-33/5/

KaterinaOsch committed:
Расставляет крупные блоки

summertime-fms commented on index.html:
```diff
@@ -7,6 +7,7 @@
     </head>
 <body>
 <header class="page-header">
+<div class="page-header-container">
```
Нужно соблюдать табуляцию и вложенность. Так код будет легче воспринимать и тебе и другим.

summertime-fms commented on index.html:
```diff
@@ -31,52 +32,58 @@
         <li class="navigation-item">
-            <a class="button" href="#">
+            <a class="navigation-link" href="#">
                 <span class="visually-hidden">
```
Лишний элемент. Нет необходимости дополнительно вкладывать текст в спан.

summertime-fms commented on index.html:
```diff
@@ -31,52 +32,58 @@
+            <span>12</span>
         </li>
         <li class="navigation-item">
             <a class="button" href="#">ХОЧУ СЮДА!</a> 
```
Верхнего регистра лучше добиваться через стили (существует свойство text-transform) А в разметке лучше писать как обычно.

summertime-fms commented on index.html:
```diff
@@ -31,52 +32,58 @@
+<div class="page-container">
+    <div class="intro">
+    <h1 class="visually-hidden">Добро пожаловать в город Седона!</h1>
+    <img src="images/welcome.svg" width="458" height="352" alt="Добро_пожаловать!">
```
В `alt` можно использовать пробелы, если что

summertime-fms commented on index.html:
```diff
@@ -31,52 +32,58 @@
-</div>
     <h2 class="city-intro">Седона — небольшой городок в Аризоне,<br>
         заслуживающий большего!</h2>
         <p class="advantages-intro">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
```
Сбилась табуляция.

summertime-fms commented on styles/styles.css:
```diff
@@ -14,7 +14,15 @@ src: url("../fonts/ptsans-700.woff2") format("woff2");
 font-display: swap;
 }
 
+html{
```
Давай пробел добавлять перед открывающей скобкой css-правила

summertime-fms commented on styles/styles.css:
```diff
@@ -14,7 +14,15 @@ src: url("../fonts/ptsans-700.woff2") format("woff2");
+}
+
 body{
+    margin:0;
```
И перед значением тоже пробел.

summertime-fms commented on styles/styles.css:
```diff
@@ -42,6 +50,28 @@ body{
+
+.page-header-container{
+    display: flex;
+    flex-direction:row;
```
Это значение по умолчанию. Его можно не прописывать.

KaterinaOsch committed:
Попытка что-то исправить

KaterinaOsch committed:
Исправляет табуляцию; не пойму, что делать с высотой "квадратиков"

Оставила фикс-высоту только для subitems. Не знаю, как вытянуть на такую же высоту, как и у остальных, нижний ряд items, все едет.
